sandbox/concurrent_bert_etl_worker2/app_all_lambdas/jobs.py

```diff
@@ -0,0 +1,131 @@
+        bert_tess_fullframe_worker_2)
+
+    s3 = boto3.resource('s3')
+    inbucket = s3.Bucket(name=os.environ.get('CACHEBUCKETNAME'))
+    bucket_name = os.environ.get('AWSBUCKETNAME')
+    bucket = s3.Bucket(name=bucket_name)
+    homedir = os.environ.get('HOME')
+
+    # NOTE: To test this, use the example below as the manual test event.
+    # DEBUG=true setting will grab it (cannot use DynamoDB but irrelevant
+    # for now).
+    #
+    # Example event:
+    # {
+    #   "id": "tess-s0001-4-1",
+    #   "xpos": 1623,
+    #   "ypos": 392,
+    #   "radius": 20,
+    #   "use_cache": "true"
+    # }
+    for event in work_queue:
+        tess_id = event['id']
+        xpos = int(event['xpos'])
+        ypos = int(event['ypos'])
+        radius = int(event['radius'])
+        pfx = f'{tess_id}/x{xpos}/y{ypos}/r{radius}'
+        basename = 'lightcurve.fits'
+        s3key = f'{pfx}/{basename}'
+        filename = os.path.join(homedir, basename)
+
+        # Use cached LC generated by previous run and skip recalculations.
+        # Skipping also means BLS Lambda listening for S3 upload will not run.
+        use_cache = event['use_cache'] == 'true'
+
+        # If this output exists and user wants to use the cache, there is
+        # nothing to do.
+        if use_cache:
+            try:
+                s3.Object(bucket_name, s3key).load()
+            except Exception:  # Does not exist
+                pass
+            else:  # It exists; nothing to do
+                ologger.info(f'{s3key} exists, skipping...')
+                continue
+
+        # f4 = np.float32, f8 = np.float64, i4 = np.int32
+        lc_tab = Table(names=('TIME', 'SAP_FLUX', 'SAP_BKG', 'QUALITY'),
+                       dtype=('f8', 'f4', 'f4', 'i4'),
+                       meta={'TELESCOP': 'TESS'})
+
+        # Grab all the light curve data points and piece them together.
+        for obj in inbucket.objects.filter(
+                Prefix=pfx, RequestPayer='requester'):
+            filename = os.path.join(homedir, os.path.basename(obj.key))
+            inbucket.download_file(
+                obj.key, filename, ExtraArgs={"RequestPayer": "requester"})
+
+            with open(filename, 'r') as fin:
+                row = fin.read().split(',')
+
+            # Clean up
+            os.remove(filename)
+
+            midtime = float(row[0])
+            signal = float(row[1])
+            background = float(row[2])
+            dqflag = int(row[3])
+
+            lc_tab.add_row((midtime, signal, background, dqflag))
+
+        # Sort table by observation time.
+        lc_tab.sort('TIME')
+
+        s = tess_id.split('-')
+        sector = int(s[1][1:])
+        camera = int(s[2])
+        ccd = int(s[3])
+        lc_tab.meta.update({
+           'CAMERA': camera,
+           'SECTOR': sector,
+           'CCD': ccd})
+
+        # TODO:
+        # Filename: tess_ticXXXX_s0001-4-1_lcc.fits ??? Want to filter by TIC ID? But then what about sector?
```

Comment:
`TIC(12digits-zero padded)/filename.fits` with TIC ID repeated in filename, along with sector. But no need to include position, radius, etc, as they are in the table header.